packages/console/src/App.tsx

```diff
@@ -1,23 +1,29 @@
 import { MetaProvider, Title } from "@solidjs/meta";
 import { Route, Router } from "@solidjs/router";
-import { Suspense } from "solid-js";
 import "./App.css";
-import { Toaster } from "./components/ui/sonner";
+import { SplashScreen } from "@/components/spash-screen";
+import Layout from "@/components/ui/layout";
+import { QueryClient, QueryClientProvider } from "@tanstack/solid-query";
+import { Suspense } from "solid-js";
 import Home from "./routes";
+export const queryClient = new QueryClient();
```

Comment:
Since it is being provided through a provider, direct access from other places must be prevented.
```suggestion
const queryClient = new QueryClient();
```